src/components/html/EcApplication.html

```diff
@@ -24,23 +39,23 @@
 
           <div>
             <p class="d-flex align-items-center">
-              <a-icon type="user" class="mr-3" />
+              <i class="fas fa-user mr-3"></i>
               {{ i19author }} : {{ author }}
             </p>
             <p class="d-flex align-items-center">
-              <a-icon type="interation" class="mr-3" />
+              <i class="fas fa-sync-alt mr-3"></i>
               {{ i19version }} : {{ version }}
             </p>
             <p class="d-flex align-items-center">
-              <a-icon type="global" class="mr-3" />
+              <i class="fa fa-globe mr-3"></i>
               <a :href="website">Website</a>
             </p>
           </div>
         </a-card>
       </a-col>
 
       <a-col :md="24" :lg="17">
-        <a-card class="w-100" :bordered="false" :tabList="tabListNoTitle" :activeTabKey="activeTabKey"
+        <a-card class="w-100" :bordered="true" :tabList="tabListNoTitle" :activeTabKey="activeTabKey"
```

Comment:
Este é o card grande, não, acho que o correto é o que estava antes mesmo:

```suggestion
        <a-card class="w-100" :bordered="false" :tabList="tabListNoTitle" :activeTabKey="activeTabKey"
```